### PR TITLE
[TMVA][SOFIE] Add basic unary operators

### PR DESCRIPTION
**Pull request by axmat**
## This Pull request:
Development of some unary operators for TMVA/SOFIE.

## Checklist:

- [x] tested changes locally

### CODE REVIEW

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2204/default, ROOT-ubuntu18.04/default with flags -Dtmva-sofie=On

File: tmva/sofie/inc/TMVA/ROperator_BasicUnary.hxx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+#include <TMVA/ROperator.hxx>
+#include <TMVA/RModel.hxx>
+#include <TMVA/SOFIE_common.hxx>
+
+namespace TMVA {
+namespace Experimental {
+namespace SOFIE {
+
+enum class EBasicUnaryOperator { kReciprocal, kSqrt };
```
Comment (sanjibansg) [inline tmva/sofie/inc/TMVA/ROperator_BasicUnary.hxx:12]: Can we include the Negation operator here as well?

cc: @lmoneta

Comment (axmat) [reply tmva/sofie/inc/TMVA/ROperator_BasicUnary.hxx]: I added the Exp operator and moved Neg to ROperator_BasicUnary as we have discussed.

**Review by lmoneta — APPROVED** [review]
LGTM. 
Only needs to fix the conflict before merging it

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2204/default, ROOT-ubuntu18.04/default

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2204/default, ROOT-ubuntu18.04/nortcxxmod

**Pull request merged**